molgenis-data-import/src/main/java/org/molgenis/data/export/mapper/AttributeMapper.java

```diff
@@ -0,0 +1,165 @@
+import static org.molgenis.data.importer.emx.EmxMetadataParser.EMX_ATTRIBUTES_UNIQUE;
+import static org.molgenis.data.importer.emx.EmxMetadataParser.EMX_ATTRIBUTES_VALIDATION_EXPRESSION;
+import static org.molgenis.data.importer.emx.EmxMetadataParser.EMX_ATTRIBUTES_VISIBLE;
+
+import com.google.common.base.Strings;
+import com.google.common.collect.Lists;
+import java.util.List;
+import java.util.Map;
+import java.util.Map.Entry;
+import java.util.stream.Collectors;
+import java.util.stream.StreamSupport;
+import org.apache.commons.lang3.StringUtils;
+import org.molgenis.data.meta.model.Attribute;
+import org.molgenis.data.meta.model.AttributeMetadata;
+import org.molgenis.data.meta.model.Tag;
+
+public class AttributeMapper {
+
+  public static final Map<String, String> ATTRIBUTE_ATTRS = newLinkedHashMap();
+
+  static {
+    ATTRIBUTE_ATTRS.put(EMX_ATTRIBUTES_NAME, AttributeMetadata.NAME);
+    ATTRIBUTE_ATTRS.put(EMX_ATTRIBUTES_LABEL, AttributeMetadata.LABEL);
+    ATTRIBUTE_ATTRS.put(EMX_ATTRIBUTES_DESCRIPTION, AttributeMetadata.DESCRIPTION);
+    ATTRIBUTE_ATTRS.put(EMX_ATTRIBUTES_ENTITY, AttributeMetadata.ENTITY);
+    ATTRIBUTE_ATTRS.put(EMX_ATTRIBUTES_DATA_TYPE, AttributeMetadata.TYPE);
+    ATTRIBUTE_ATTRS.put(EMX_ATTRIBUTES_REF_ENTITY, AttributeMetadata.REF_ENTITY_TYPE);
+    ATTRIBUTE_ATTRS.put(EMX_ATTRIBUTES_NILLABLE, AttributeMetadata.IS_NULLABLE);
+    ATTRIBUTE_ATTRS.put(EMX_ATTRIBUTES_UNIQUE, AttributeMetadata.IS_UNIQUE);
+    ATTRIBUTE_ATTRS.put(EMX_ATTRIBUTES_VISIBLE, AttributeMetadata.IS_VISIBLE);
+    ATTRIBUTE_ATTRS.put(EMX_ATTRIBUTES_ID_ATTRIBUTE, AttributeMetadata.IS_ID_ATTRIBUTE);
+    ATTRIBUTE_ATTRS.put(EMX_ATTRIBUTES_LABEL_ATTRIBUTE, AttributeMetadata.IS_LABEL_ATTRIBUTE);
+    ATTRIBUTE_ATTRS.put(EMX_ATTRIBUTES_READ_ONLY, AttributeMetadata.IS_READ_ONLY);
+    ATTRIBUTE_ATTRS.put(EMX_ATTRIBUTES_AGGREGATEABLE, AttributeMetadata.IS_AGGREGATABLE);
+    ATTRIBUTE_ATTRS.put(EMX_ATTRIBUTES_LOOKUP_ATTRIBUTE, AttributeMetadata.LOOKUP_ATTRIBUTE_INDEX);
+    ATTRIBUTE_ATTRS.put(EMX_ATTRIBUTES_ENUM_OPTIONS, AttributeMetadata.ENUM_OPTIONS);
+    ATTRIBUTE_ATTRS.put(EMX_ATTRIBUTES_PART_OF_ATTRIBUTE, AttributeMetadata.PARENT);
+    ATTRIBUTE_ATTRS.put(EMX_ATTRIBUTES_RANGE_MAX, AttributeMetadata.RANGE_MAX);
+    ATTRIBUTE_ATTRS.put(EMX_ATTRIBUTES_RANGE_MIN, AttributeMetadata.RANGE_MIN);
+    ATTRIBUTE_ATTRS.put(EMX_ATTRIBUTES_MAPPED_BY, AttributeMetadata.MAPPED_BY);
+    ATTRIBUTE_ATTRS.put(EMX_ATTRIBUTES_EXPRESSION, AttributeMetadata.EXPRESSION);
+    ATTRIBUTE_ATTRS.put(
+        EMX_ATTRIBUTES_VALIDATION_EXPRESSION, AttributeMetadata.VALIDATION_EXPRESSION);
+    ATTRIBUTE_ATTRS.put(EMX_ATTRIBUTES_DEFAULT_VALUE, AttributeMetadata.DEFAULT_VALUE);
+    ATTRIBUTE_ATTRS.put(EMX_ATTRIBUTES_TAGS, AttributeMetadata.TAGS);
+  }
+
+  private AttributeMapper() {}
+
+  public static List<Object> map(Attribute attr) {
+    List<Object> row = Lists.newArrayList();
```

Comment:
```suggestion
    List<Object> row = new ArrayList<>();
```
See newArrayList documentation:
```
   * <p><b>Note for Java 7 and later:</b> this method is now unnecessary and should be treated as
   * deprecated. Instead, use the {@code ArrayList} {@linkplain ArrayList#ArrayList() constructor}
   * directly, taking advantage of the new <a href="http://goo.gl/iz2Wi">"diamond" syntax</a>.
```